### ISSUE TITLE
Auto Shotgun is overpowered in pretty much every way.

### ISSUE THREAD

**Issue by StruggleDuck** [Regalis11/Barotrauma#7538]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
As it is now, the Auto shotgun is easily one of the most overpowered items in the entire game. It currently has the ability to infinitely stun lock anyone unlucky enough to cross its path, and on top of that, the Auto shotguns DPS is one of the highest of any weapon in the game, even capable of doing more damage per second that a coil gun using default ammo against armored targets (while using explosive slugs). When it comes to un-armored targets, the Auto Shotgun does more DPS than almost every sub-mounted weapon, only being outdone by explosive coilgun ammo, and Pulse Tri-laser (if all 3 beams hit the target). On top of all this, the autoshotgun has the largest ammunition capacities of any handheld gun, with it being capable of holding 32 shots. Its total damage per mag is somewhere around 5x higher than every other handheld weapon. The autoshotguns ammuniton is not even that expensive, requiring only 1 incendium and 1 plastic for 6 shells on average, or for the regular round only 1 lead, magnesium and 1 plastic for another 6 rounds. When compared to the assault rifle, which requires 1 physicorium per 20 bullets (as well as 1 Fe+ and 1 Mg). 

**Version** 
v0.15.13.0

**Comment by StruggleDuck** [comment]
It's 6 on average when taking into account the munitions expert perk, which is what allows you to make double the ammunition 50% of the time. Without that perk you can't actually make explosive slugs

**Comment by KerbalCitizen** [comment]
Assuming the 50% ammo applies to assault rifle (not sure if the fabricator handles that right), then it's really 30 shots. 30 shots each doing 30 damage is 900 damage. 6 shotguns shells each doing 64 damage (assuming all 8 pellets hit) do 384 damage. Roughly speaking, 3 shotgun shell crafts do the same amount of damage as one assault rifle magazine craft. Alien ruins can net you quite a bit of physicorium. If you bought the ingredients for both shotgun shells and assault rifle magazines, I think the price per damage would be fairly close.

**Comment by StruggleDuck** [comment]
I think its more so the fact that its super easy to obtain the materials for shotgun shells, as you can just buy them at an outpost instead of going out of your way to go into a ruin. plus, chances are that you are going to need to expend more ammo on killing whatever is inside the ruin.

**Comment by KerbalCitizen** [comment]
I've only done ~5 alien ruins before and never got past European Ridge biome, so not a huge amount of experience, but I have only seen a hostile fractal guardian once. Also seen a husk and lots of red cells, but that's it. I got the fractal guardian to less than half health with just one harpoon gun, so an assault rifle mag (20 shots vs 6) should easily kill it. Less than one physicorium bar spent on fighting the hardest enemy I've encountered so far in ruins. And an entire ruin can net much more physicorium, I'd say about 8 bars of it. 
Some outposts do sell physicorium bars, which even at ~170mk is price competitive with buying shotgun shells/ components to make shotgun shells. Remember about 3 shotgun shell crafts (18 shells) does about the same damage as one assault rifle craft (30 bullets). 

Let's analyze just buying 18 shells. Wiki says 50mk or 38mk to buy a shotgun shell, so let's use 40mk as the number. You end up with a bill of 720mk if you bought that much. 

Buying the components of 3 shotgun shell crafts means 3 Mg, 3 lead, and 3 plastic. Plastic is usually already on sub, so let's just ignore it. Lead and Mg cost ~30mk when they are sold, but generally it's quite rare to find it sold. Lead and Mg is only sold at mining, About 180mk or so. 

Crafting one assault rifle mag costs 1 Mg, 1 steel, and 1 physicorium. Mg and steel amount to ~50mk if you purchase. Physicorium bar is ~190mk, sold at mining and research. So 240mk in total. 

In conclusion, straight up buying shotgun shells is very bad in terms of cost for damage. Buying the ingredients for shotgun shells is quite good for cost effectiveness, but relatively rare ingredients from purchasing. Buying the ingredients for assault rifle mag craft is better, since only one Mg used and physicorium is sold at more outposts (found in great quantity at alien ruins anyways).

**Comment by StruggleDuck** [comment]
even still, the main point is that shotgun ammo is much more available, phycorium can only be bought at stations 1 at a time, while with shotgun ammo you can buy enough mats to last you multiple runs before you run out of ammo

**Comment by itchyOwl** [comment]
Related internal ticket Regalis11/Barotrauma-development#3233.

**Comment by Rokvach** [comment]
Tested, no issues. closing.
